andela/ah-the-jedi-backend
# #165305255 Integrate TravisCI with readme badge.

#### What does this PR do?
Integrate TravisCI with the project and include TravisCI badge in Readme.MD file.
#### Description of Task to be completed?
- Add the project repository to Travis CI repositories.
- Create `.travis.yml` file with relevant settings and commands.
- Push the branch to Github.
- Wait for the build of the branch to be complete.
#### How should this be manually tested?
- Clone the repository: `git clone https://github.com/andela/ah-the-jedi-backend.git`
- Fetch origin to get all branches: `git fetch origin`
- Checkout to the working branch  `git checkout ch-integrate-travisCI-badge-165305255`
- Check TravisCI webpage for build status: link [here](https://travis-ci.com/andela/ah-the-jedi-backend)
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#165305255](https://www.pivotaltracker.com/story/show/165305255)
#### Screenshots (if appropriate)
#### Image of successful build on travis.
![image](https://user-images.githubusercontent.com/17095461/56582780-be390b80-65e0-11e9-834d-8d8c29bf82c6.png)
